clouddrift/adapters/subsurface.py

```diff
@@ -0,0 +1,170 @@
+"""
+This module defines functions used to adapt the subsurface float trajectories as
+a ragged-array dataset.
```

Comment:
Tiny bit more info here about the dataset perhaps?

Reply:
rename to subsurfacefloat.py?

Reply:
Does this dataset have a proper name? It's not clear to me from its website. If not, I suggest `subsurface_floats`.

Reply:
That was also my hesitation. I don't see any real defined name for it..

Reply:
I added _some_ details. Let me know if you think this should be further improved.